### PR TITLE
Enhancement #56: Fix ownership of layers.conf file

### PR DESCRIPTION
**Pull request by Bejusek**
This PR fixes issue #56 .

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/coi-gov-pl/puppet-jboss/pull/57" target="_blank">ReviewNinja

**Comment by cardil** [comment]
@Bejusek could you write some rspec-puppet test to cover your change?

BTW. acceptance tests are still failing due to misbehaving Travis: travis-ci/travis-ci#5578

**Comment by Bejusek** [comment]
@coi-gov-pl/jboss-puppet-team please review.

File: manifests/internal/module/registerlayer.pp [hunk 1]
```diff
@@ -14,6 +14,7 @@
     exec { "jboss::module::layer::${layer}":
       command => "/bin/awk -F'=' 'BEGIN {ins = 0} /^layers=/ { ins = ins + 1; print \$1=${layer},\$2 } END {if(ins == 0) print \"layers=${layer},base\"}' > ${jboss::home}/modules/layers.conf",
       unless  => "/bin/egrep -e '^layers=.*${layer}.*' ${jboss::home}/modules/layers.conf",
```
Comment (cardil) [inline manifests/internal/module/registerlayer.pp]: If you are adding test for register layer, which is cool, you can also verify crucial command and unless parameters !resolve

Comment (cardil) [inline manifests/internal/module/registerlayer.pp]: !resolved

**Comment by cardil** [comment]
Nice. :star: from me!

**Comment by Bejusek** [comment]
@coi-gov-pl/jboss-puppet-team please review so we can finally merge this.

**Comment by Cosaquee** [comment]
@Bejusek feel free to merge it

**Comment by Bejusek** [comment]
Merge fixes #56

**Pull request merged**